debugtools/DDR_VM/generate.xml

```diff
@@ -172,13 +172,11 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			</then>
 			<else>
 				<shellscript shell="bash" dir="${TMP_DIR}">
-					mkdir ${TMP_DIR}/debugtools
-					cp -af ${WORKSPACE}/debugtools/DDR_* ${TMP_DIR}/debugtools
-					cp -af ${WORKSPACE}/jcl ${TMP_DIR}
-					cp -af ${WORKSPACE}/../build/RAS_Binaries/*.jar ${TMP_DIR}/RAS_Binaries
-					mkdir ${TMP_DIR}/sourcetools/
-					cp -af ${WORKSPACE}/sourcetools/lib ${TMP_DIR}/sourcetools/
-					cp -af ${WORKSPACE}/sourcetools/*.mk ${TMP_DIR}/sourcetools/
+					mkdir -p debugtools/ RAS_Binaries/ sourcetools/
+					cp -af ${WORKSPACE}/jcl ./
```

Comment:
This appears to be a change in behavior, as the original code copied to ${TMP_DIR}. Why the change?

Reply:
It's not a change in behavior: the commands are executed in the directory named `${TMP_DIR}`.

Reply:
Ok, but originally the script wasn't dependent on the cd, and now it is. This doesn't seem like an improvement.

Reply:
I can add seven uses of `${TMP_DIR}` if you feel strongly, but, in my opinion, _that_ would not be an improvement.

Reply:
Ok, I see what you mean.